components/account/NotificationsPanel.tsx

```
@@ -0,0 +1,382 @@
+        </div>
+      </div>
+      <div>
+        <h3 className="text-sm font-semibold text-white">
+          You have {alert.markets.length} redeemable markets
+        </h3>
+      </div>
+    </AlertCard>
+  );
+};
+
+const RelevantMarketDisputeItem = ({}: {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)